src/Entity/OrganizationRepository.php

```diff
@@ -4,7 +4,29 @@
 namespace AppBundle\Entity;
 
 use Doctrine\ORM\EntityRepository;
-
+/**
+ * @extends EntityRepository<Organization>
+ */
 class OrganizationRepository extends EntityRepository
 {
+
+    public function reverseFindByOrganization(Organization $org): LocalBusiness|Store|null
+    {
+        $query = $this->createQueryBuilder('o')
+        ->select('s', 'r')
+        ->leftJoin(Store::class, 's', 'WITH', 's.organization = o.id')
+        ->leftJoin(LocalBusiness::class, 'r', 'WITH', 'r.organization = o.id')
+        ->setMaxResults(1)
+        ->where('o = :org')
+        ->setParameter('org', $org)
+        ->getQuery()
+        ->getResult();
```

Comment:
Why not use `getOneOrNullResult`?

Reply:
When i'm using this method the following exception is raised
> More than one result was found for query although one row or none was expected.